_codux/boards/avatar/avatar.board.tsx

```diff
@@ -0,0 +1,23 @@
QTN5HYKTyK2nvn0dj0OYMZmp30F0CP06Zo7gQLvuGJsn2iMpvOPLRe8/op/tPjVOm9/io5h8MkCFdZ2UxMqnw4glpXTX0x88buvxTNgc1FjyRF9Hg9WwwngUDOTJEs9oYVr3hANMR2B54+lwsgGv7jCkM6hvq+Co4HUs6gdpiq36gYuclzH04VZLeaBB1Wdj5AX2DqaUfxdEKgVZK8WEjY+Lg5VR1rt0YmpoM+DqUKGivPf1boZ0K+mDdMby38p2MkqaXhT1/mb+tpYjzIy4BnY41vq4XYBDsEia56JabGakztumCrzqb7sDAsM30SoWWxXOQJ3qsVNw/eIiRcrHfdSPXifs4cp26UI1wtW5Jinq49JLX4e/fZ5jFuGs5NgNTN+OUfHDC5tsH2Scmah1eRfWPt+uHC65y4iwMIbFIFaGbIGF/IXo0G6jl1YyFS9jbkv1Vyr0axFo8yyu7hE2xnYgy2vbZBHiaixL6Rd969b2eaJySgKNS8fTxa9mTf7XtSEhA4g+wQWjCgvvBIwM3gmJ+OQh0nKExsAudH4H/4imiE8Vhdj8O9l1Db5YbNf2rW5Wfzcefr5+Qlo8ib+G2i7eqeODqW5DUbOb0jBpNjTRtMIrUEodqs1GgWiRewyMNF120XgxuMO2l9hFKX29GOmyrLtXTyto2V7z9fsPjeCB1BbmISUOddnP7Sa9U5FM3r9PEM16mX6opqRgHHKBSa+nLfCHMZS+ZNS+/uLRsrW3TNXeuNA/FvhgYkd8AlURj/G/rMp4y5MvGrSy5kTEmWVBAKU8S8kyOJvhuZoGRmDbil60SiHap1tkSKxfl6pNvaXVXU/2TvV40bBQrbHJKxnjKMvMrfwYZcJoi2z0Zq0D/a51r03MrjPBPFFITpezoYg1G+xCHhZFOgk7U+CAFvHyg61DLST7gRiwvBWLim7MUwbxLLswBkh9qbKqTRXv4lBiR2aBESL7m4eYs20lnMmYlqKUS1v+zJgwCPB9VdY287d60S3SilnSFe3KfaXuTqVGfQiNGuS1lEqwijzcWHQP+2Hw8q8ODIcjEMwlqqqDx/3ks9jvRa7aQF+Hpnl7N8K+2OHlUfgFsJol7TCh/x+OaIqc9rMm1F0PTsqYYhR4cXZIx4lTqDnZSmJQ7PYl9ALY/wZGhrDpbboFafqcAvhRqqg+FTTP+Q4xR98NHCG6bcrMjXQaWkLSSJBEbniVAUM0o3fxC6h6diR7GLQebMKp/rG3nRc7P0WF1CYwqo1wZFLHU9cdgV1aWqDWhagUe82FZLPwiaC6IoDXdYaICfDVF2P4ZTQ/Ggwz0CDGTLlzuhprbm9YCnVKv4F6ww3qzBu2eU4uX+uTjKFKRwV1VC5S5PdwGK2G6UZGMrpsYBMyWhyhA3NZQ+umgh6Nu+nKQrruNXc/QrDC25VqKAe8fslky1LPDEiqGHX0X7eeHK5+IBik3PIM0J4CrVIOgUPJbp+TujxfCcxPl06ifSV6vTHmvea74hMZzSRHLkCqQpYKpRbVUX7kFbfFrno1MV+A7pPvk8ULX+ZN5zbqkdVe7enck0QCz9urAvkBm/TJm6aFO2BK/JAJelcKPKPJD9LO0IB2UQQb6mPkeaVEDGbO8WyjlS3Q8JXUdiFXY81vCBMnfOfl3JYDH+K7X4ALfvJQBr/KAZsS5k0XoI7LXfaNhrq00tgZU30ABDu1RdTh3aHE1tzKxsDKOWegt0n0fkkthEr9y0OxlUol2t+8+hdwN5LQYsx+qWT7wduOdq+6nNFTDGIfqNGIs6UO9c00LqMOPiecMZv/oVbGUpu3Qy4VcQ9ABc1aCjQQj14DIBliGN31hme7egwBc9dls+FE1KkMMGz2xmnUXBD+N3x+eftYOexpBkJxvK4hDYTwxsMtzBE80EyeSgtWmDJdXSeTz73Tqab+j1TWB+R3Q7bgFnOzU0v6zxiIdV/lcxQtT6nn5FFCS2EhIvCPECbPOAKk+D+a49pqN+s97kyqTI5LjuXu32ky6d41JI3YUXJCfxL4mFZAuk38rjm/nnbrw75IHIfXHEX7BQvl8FblX9XeFj+MHHNV8r9JYQ0e/h7G7+U1h42bpudrdOLDJnjFcsXTjnv+Hd1vGAWEW+VCJfrRpGcT3yEMSmWWcYZLhQq/ZS5HQzbdO4FjJ3CkloyLOuJlpt4o0OB+Ln13n/aIbTQJaKPLEDCN+r7EPWf16qA9pQ1AkCIM4Yd68MkBnX8vO2Ymy96twoBytRTTeKqboVMd+gni1Ih6ZHjCizfpVVZ3Arb0+m2DbKasot6HTYR+gU4+uSOFy5V7qEOOS5jp8zlhqxO8KkihraZuT2IYIzd2BARFHOtIw9ARciNxSkU+UQPGcAc4eo2gybAtP+ACOg/f+AXVuLyUujuavzdmUGWw5df/tB+DQyAVh9zFC7ZewT6rxmN12wybWDpS3av6dAFf8LYP7yDMFH7hgMGgApVnAz40ENUbsQ70H9MSMkBS5pWc3//1Mv88q82W6v6DojM7EKnxCvsb5GDzwxpb7qfErVO7fp8RZ9w/u1sSXkcw91qvtmVpQTEpGwrGadgNfUnKWN9ZjvJan8mMuWV9dv9KPzS2JqyD+YFYTPUG2lJ86281mW5xiJ44KvS04qi0YCruV00B+hxio8PA5vzEkTkvwr7DWdIAjSZyTqpJSfo+HTOEx7XzmtUZ9TEHeZ2IAsc+1PBbS0FZzzPm2P2KUBTNKWNweg7HjkAAAAAEVYSUa6AAAARXhpZgAASUkqAAgAAAAGABIBAwABAAAAAQAAABoBBQABAAAAVgAAABsBBQABAAAAXgAAACgBAwABAAAAAgAAABMCAwABAAAAAQAAAGmHBAABAAAAZgAAAAAAAAA4YwAA6AMAADhjAADoAwAABgAAkAcABAAAADAyMTABkQcABAAAAAECAwAAoAcABAAAADAxMDABoAMAAQAAAP//AAACoAQAAQAAAKgBAAADoAQAAQAAAIQBAAAAAAAA`;
```

Comment:
Why? Just use a URL.

Reply:
because URL might gone? with data URI board will be always available.